blankiety_galantow/core/game_master.py

```diff
@@ -20,10 +18,12 @@ class GameMaster:
     black_card: BlackCard
 
     # FIXME: Zrezygnować z callbacku na rzecz czegoś "ładniejszego"
-    def __init__(self, players: ObservableList, chat: Chat, settings: RoomSettings, players_update_callback):
-        self.players = players
-        self.chat = chat
-        self.settings = settings
+    def __init__(self, room: 'Room'):
+        self.room = room
+        self.players = room.players
+        self.chat = room.chat
+        self.settings = room.settings
+        self.players_update_callback = room.send_players_update
```

Comment:
Zmienić nazwę funkcji `players_update_callback` na `update_players` albo `send_players_update`, bo technicznie, to już nie jest stricte callback.